relay-base-schema/src/events.rs

```diff
@@ -41,6 +41,9 @@ pub enum EventType {
     ExpectStaple,
     /// Performance monitoring transactions carrying spans.
     Transaction,
+    /// User feedback payload. TODO(Jferg): Change this to UserFeedback once old UserReport
+    /// logic is deprecated.
+    UserReportV2,
```

Comment:
As per Relay's coding guidelines, documentation should have one line followed by additional paragraphs:
```suggestion
    /// User feedback payload.
    ///
    /// TODO(Jferg): Change this to UserFeedback once old UserReport logic is deprecated.
    UserReportV2,
```